ext/CreateParametersExt.jl

```diff
@@ -319,12 +321,21 @@ function EnergyHydrologyParameters(
     K_sat::F,
     S_s::F,
     θ_r::F,
-    PAR_albedo::SF = 0.2,
-    NIR_albedo::SF = 0.4,
+    PAR_albedo_dry::SF = 0.2,
+    NIR_albedo_dry::SF = 0.4,
+    PAR_albedo_wet::SF = 0.2,
+    NIR_albedo_wet::SF = 0.4,
+    ###########DEPRECATED################
+    PAR_albedo::SFD = nothing,
+    NIR_albedo::SFD = nothing,
+    ####################################
```

Comment:
Should this be marked as deprecated, or is support going to be continued?

Reply:
When the formula recomputes albedo, if both wet and dry are the same, then the calculated value is also that same value. For example if a user passes in `PAR_albedo = 0.2`, then the constructor sets `PAR_albedo_dry = 0.2` and `PAR_albedo_wet = 0.2`. Then when calculating albedo, the `p.soil.PAR_albedo` is set to 
`PAR_albedo_wet * \theta + PAR_albedo_dry * (1-\theta) = 0.2`. So isn't the input albedo the same as the one in the cache?

Reply:
One idea (would have to be a follow-on PR) is to make an AbstractSoilAlbedoModel with two concrete types, one with wet/dry and the other with the bulk (constant) values. Each struct could store their respective values. Then we could also move the albedo parameters outside of EnergyHydrologyParameters and into the AtmosDrivenBC struct.

Reply:
the top thickness parameter could also live in the struct

Reply:
the update_albedo function could then dispatch off of BC type and internally on albedo model type.

Reply:
> When the formula recomputes albedo, if both wet and dry are the same, then the calculated value is also that same value. For example if a user passes in PAR_albedo = 0.2, then the constructor sets PAR_albedo_dry = 0.2 and PAR_albedo_wet = 0.2. Then when calculating albedo, the p.soil.PAR_albedo is set to
> PAR_albedo_wet * \theta + PAR_albedo_dry * (1-\theta) = 0.2. So isn't the input albedo the same as the one in the cache?

Okay! I thought the integral would change things.